README.md

```diff
@@ -4,47 +4,109 @@
 uftrace
 =======
 
-The uftrace tool is to trace and analyze execution of a program
-written in C/C++.  It was heavily inspired by the ftrace framework
-of the Linux kernel (especially function graph tracer) and supports
-userspace programs.  It supports various kind of commands and filters
-to help analysis of the program execution and performance.
+uftrace is a function call graph tracer for C, C++, Rust and Python programs.
+
+uftrace traces, records and analyzes the execution of programs written in
+C, C++ and Rust. It was heavily inspired by the ftrace framework of the
```

Comment:
And Python, but I think this is redundant.

Reply:
I've replaced this paragraph now with the text from https://www.blackhat.com/asia-23/arsenal/schedule/#uftrace-dynamic-analysistracing-tool-for-ccrust-programs-30951, which is even better.